stdlib/universe/moving_average.go

```diff
@@ -0,0 +1,585 @@
+package universe
+
+import (
+	"container/list"
+	"fmt"
```

Comment:
Can you separate `github.com` import from others with a newline?

Reply:
Please, address this.

Reply:
did we add the newline here?